Fix graphviz out-of-source build.

This was already accepted into future versions:
https://gitlab.com/graphviz/graphviz/issues/1596

--- graphviz-2.42.0/tclpkg/tcldot/Makefile.am
+++ graphviz-2.42.0/tclpkg/tcldot/Makefile.am
@@ -7,7 +7,7 @@ SUBDIRS = demo
 TCLHANDLE_INCLUDES=-I$(top_srcdir)/tclpkg/tclhandle
 TCLHANDLE_LIBS=$(top_builddir)/tclpkg/tclhandle/libtclhandle_C.la
 if WITH_LIBGD
-GDTCLFT_LIBS = $(top_srcdir)/tclpkg/gdtclft/libgdtclft_C.la
+GDTCLFT_LIBS = $(top_builddir)/tclpkg/gdtclft/libgdtclft_C.la
 endif
 
 pdfdir = $(pkgdatadir)/doc/pdf
